tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -253,10 +272,10 @@ filings.forEach((filing: any, index: number) => {
       } else if (item.isFutureEffectiveAlterationPending) {
         expect(wrapper.findComponent(FutureEffectivePending).exists()).toBe(true)
       } else if (item.isFutureEffectiveAlteration) {
-        expect(wrapper.ffindComponentind(FutureEffective).exists()).toBe(true)
```

Comment:
This didn't cause the test to fail because we don't (yet) test future effective alterations.